src/services/UniversityService.java

```diff
@@ -0,0 +1,85 @@
+package services;
+
+import main.Faculty;
+import main.University;
+import java.util.Scanner;
+
+public class UniversityService {
+    private static Scanner scanner = new Scanner(System.in);
+
+    public static void initializeFaculties(int numberOfFaculties, University university) {
+        university.setFaculties(new Faculty[numberOfFaculties]);
+        Faculty[] faculties = university.getFaculties();
+        for (int i = 0; i < numberOfFaculties ; i++) {
+            System.out.println("Please enter the name of faculty " + (i + 1) + ": ");
+            faculties[i] = new Faculty();
+            faculties[i].setNameOfFaculty(scanner.next());
+            System.out.print("Please enter the number of groups in faculty " + faculties[i].getNameOfFaculty() + ": ");
+            FacultyService.initializeGroups(InputController.checkForWrongInput("groups"),faculties[i]);
+        }
+    }
+
+    public static double getAveragePointOfStudent(String nameOfFaculty, int numberOfGroup, String nameOfStudent,Faculty[] faculties){
+        int indexOfNecessaryFaculty = 0;
+
+        for (int i = 0; i < faculties.length; i++) {
+            if (faculties[i].getNameOfFaculty().equals(nameOfFaculty)) {
+                indexOfNecessaryFaculty = i;
+                break;
+            }
+        }
+        return FacultyService.getAveragePointOfStudent(numberOfGroup,nameOfStudent,faculties[indexOfNecessaryFaculty].getGroups());
```

Comment:
Display average grade rounded to two decimal places. You may want to use BigDecimal, as in https://stackoverflow.com/questions/2808535/round-a-double-to-2-decimal-places